src/i_winmusic.c

```diff
@@ -1451,6 +1451,11 @@ static void I_WIN_ShutdownMusic(void)
 
     if (buffer.data)
     {
+        while (MidiStreamHdr.dwFlags & MHDR_INQUEUE)
```

Comment:
Do we risk an infinite loop here?

Reply:
I'd rather have an additional e.g. "at most 1 second" condition to be honest.

Reply:
There doesn't seem to be any side effect to just forcing the flag off to get past `midiOutUnprepareHeader()` and we're shutting down the module anyway, so https://github.com/fabiangreffrath/woof/pull/856/commits/9546ca265d5d78106846ac3ea8d667269132ef8c should be fine.